### DISCUSSION TITLE
Interesting css

### DISCUSSION THREAD

**Discussion by kyrapieterse** [maxi-blocks/maxi-blocks#4592] [Announcements]
### Description

https://twitter.com/thatroblennon/status/1635399234626416640?t=9HBdQ2fcsyXRBq85uoWLZA&s=19

**Comment by Marko64** [comment]
Maybe move it to discussions?

**Comment by Naaaaiix** [comment]
Are we interest on adding it into Maxi? Should be fast

**Comment by kyrapieterse** [comment]
Looks cool, lonely items are a pain
